raiden-dapp/src/views/OpenChannelRoute.vue

```diff
@@ -109,7 +109,7 @@ export default class OpenChannelRoute extends Mixins(NavigationMixin) {
     return this.getToken(address) || ({ address } as Token);
   }
 
-  beforeRouteLeave(to: Route, from: Route, next: NavigationGuardNext) {
+  beforeRouteLeave(_to: Route, _from: Route, next: NavigationGuardNext) {
```

Comment:
Hmm. That wasn't intentionally made. So it must have been some formatting. Which makes me wonder if we had updated your formatter versions? But in fact it is correct because the parameters are not used and tells TypeScript to not care and ignore.